views/window/dockedwindow.html

```diff
@@ -1,47 +1,47 @@
 <div class="docked-window d-flex">
-    <div class="modal-content d-flex flex-column flex-fill" ng-class="{'bg-transparent': overlay, 'border-0': !border}">
-      <div class="c-docked-window__buttons">
-        <span class="text-truncate flex-fill" title="{{label}}">
-          <i ng-if="icon" ng-class="icon" class="mr-1"></i>
-          <span ng-if="iconHtml" ng-bind-html="iconHtml"></span>
-          {{label}}
-        </span>
+  <div class="modal-content d-flex flex-column flex-fill" ng-class="{'bg-transparent': overlay, 'border-0': !border}">
+    <div class="c-docked-window__buttons">
+      <span class="text-truncate flex-fill" title="{{label}}">
+        <i ng-if="icon" ng-class="icon" class="mr-1"></i>
+        <span ng-if="iconHtml" ng-bind-html="iconHtml"></span>
+        {{label}}
+      </span>
 
-        <span ng-if="helpContext" class="d-inline-flex">
-          <context-onboarding context="{{helpContext}}"></context-onboarding>
-        </span>
+      <span ng-if="helpContext" class="d-inline-flex">
+        <context-onboarding context="{{helpContext}}"></context-onboarding>
+      </span>
 
-        <span ng-repeat="headerBtnCfg in headerBtns | reverse"
-            ng-click="windowCtrl.onHeaderBtnClick($event, headerBtnCfg);"
-            title="{{headerBtnCfg.tooltip}}">
-          <i class="{{'c-glyph ' + headerBtnCfg.iconClass}}"></i>
-        </span>
+      <span ng-repeat="headerBtnCfg in headerBtns | reverse"
```

Comment:
Did these ones not complain too? Was the error because I had an accidental (and invalid) type="button" on that one span?

Reply:
No, this one doesn't have `aria-label` on it, though it probably should have one that matches `title`. That would produce the same error unless it was also switched to a `button`.